.gitignore

```diff
@@ -1,17 +1,11 @@
-.DS_Store
-examples/.terraform
-examples
 bin
 
-# Local .terraform directories
-**/.terraform/*
-
-# .tfstate files
-*.tfstate
-*.tfstate.*
-
-# Crash log files
-crash.log
+# Local terraform testing
+/main.tf
+/main.json
+/.terraform*
+/*.tfstate*
+/crash.log
```

Comment:
Note: please don't encourage testing from the project root; _`itsdangerous`_

Reply:
An unspoken rule dictates that testing must be performed in `examples`, not because of the name (attrociusly misused), but just because it's a subdirectory and side effects are confined there.

Reply:
```suggestion
/scratchpad
```

```suggestion
/testing
```

Reply:
I didn't add these entries, just consolidated/tidied them. I'm fine with scrubbing them but would check with @DavidGOrtega :/

Reply:
🤦🏼‍♂️ That's what happens when I read a partial diff thinking that it's the whole file. Sorry!

Reply:
It's fine to leave them to prevent further accidents, but... 🙃

Reply:
feel free to open a follow-up issue/PR :)